### PR TITLE
add assignment 2 solution

### PR DESCRIPTION
**Pull request by purisaurabh**
Solution for javascript assignment 2

### CODE REVIEW

File: Javascript_Assignment_2/Question_3/arr_index.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+// iterating through an array
+constArr.push("Flask")
+constArr.push("ios")
+for(let i = 0; i < constArr.length ; i++){
+    console.log(constArr[i]);
+}
+
+// array concatenation
+let otherConstCourses = ["Node.js", "Expess.js"];
+let newConstArr = letArr.concat(otherConstCourses)
```
Comment (Rushikesh-josh) [inline Javascript_Assignment_2/Question_3/arr_index.js:114]: is there any way to concat array without using concat fun?

File: Javascript_Assignment_2/Question_3/arr_index.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+letArr[1] = "Flutter"
+console.log("After modifying the array : ", letArr);
+
+// Adding the element into an array
+letArr.push("Flask")
+console.log("The letArr after pushing into the last index : ", letArr);
+
+// removing the element from the arr
+letArr.pop()
+console.log("After removing the last element from the array" , letArr);
```
Comment (Rushikesh-josh) [inline Javascript_Assignment_2/Question_3/arr_index.js:27]: Will you please write the output after each console log?

Comment (purisaurabh) [reply Javascript_Assignment_2/Question_3/arr_index.js:38]: ok

File: Javascript_Assignment_2/Question_3/arr_index.js (hunk 1 above)
Comment (purisaurabh) [reply Javascript_Assignment_2/Question_3/arr_index.js:216]: ok

Comment (Rushikesh-josh) [reply Javascript_Assignment_2/Question_3/arr_index.js:216]: here you should write yes we can use spread operator

**Review by Rushikesh-josh — APPROVED** [review]

**Pull request merged**